contracts/src/interfaces/IHyperdriveAdminController.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.22;
+
+interface IHyperdriveAdminController {
+    /// @notice Returns the Hyperdrive governance address.
+    /// @return The Hyperdrive governance address.
+    function hyperdriveGovernance() external view returns (address);
+
+    /// @notice Returns the fee collector that is the target of fee collections.
+    /// @return The fee collector.
+    function feeCollector() external view returns (address);
+
+    /// @notice Returns the sweep collector that can sweep stuck tokens from
+    ///         Hyperdrive instances.
+    /// @return The sweep collector.
+    function sweepCollector() external view returns (address);
+
+    /// @notice Returns the checkpoint rewarder that can pay out rewards to
+    ///         checkpoint minters.
+    /// @return The checkpoint rewarder.
+    function checkpointRewarder() external view returns (address);
+
+    // TODO: A better interface would be `isPauser`, but this can't be changed
+    //       without swapping out the factory.
```

Comment:
Why would isPauser be better?

Reply:
It would be more efficient. Right now, we have to scan through the full list of pausers to check if an address is a pauser. This places constraints on the amount of pausers that the factory can have.